hamlpy/hamlpy_watcher.py

```diff
@@ -129,14 +143,19 @@ def _watch_folder(folder, destination, compiler_args):
                 if (not fullpath in compiled or
                     compiled[fullpath] < mtime or
                     not os.path.isfile(compiled_path)):
-                    compile_file(fullpath, compiled_path, compiler_args)
+                    if not compile_file(fullpath, compiled_path, compiler_args):
```

Comment:
That's unrelated to this PR but the indentation of the previous line makes it really weird to read.